src/utils/useFiltered_SortedPeople.ts

```diff
@@ -0,0 +1,65 @@
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+
+export const useFilteredSortedPeople = (
+  people: Person[],
+): {
+  sortedPeople: Person[];
+  order: string | null;
+  typeSort: string | null;
+} => {
+  const [searchParams] = useSearchParams();
+  const selectedCenturies = searchParams.getAll('centuries').map(item => +item);
```

Comment:
I recommend using the existing transformation, cuz it's more readable.

u can use `Number` or `parseInt` for it

+ - looks like a mathematical operation
```suggestion
  const selectedCenturies = searchParams.getAll('centuries').map(item => Number(item));
```